src/remoteConnector.ts

```diff
@@ -708,6 +708,7 @@ export default class RemoteConnector extends Disposable {
 
 				if (password) {
 					await this.showSSHPasswordModal(password, params.gitpodHost);
+					this.telemetry.sendRawTelemetryEvent('vscode_desktop_ssh', { kind: 'gateway', status: 'connecting', ...params, gitpodVersion: gitpodVersion.raw, copy: true });
```

Comment:
I don't think we should reuse `vscode_desktop_ssh`,

but instead have a new event like `vscode_desktop_ssh_gateway_modal` with all metadata as for `vscode_desktop_ssh` event + `action` value plus `cancelled` for undefined.